src/main/java/com/sublinks/sublinksapi/slurfilter/services/SlurFilterService.java

```diff
@@ -9,6 +9,8 @@
 import java.util.List;
```

Comment:
The changes in this file are unrelated to the original issue. I think they should be moved to a new issue. I have an idea on how to better implement the slur filter lookup.